src/Transport/RestTransport.php

```diff
@@ -153,6 +155,12 @@ public function startServerStreamingCall(Call $call, array $options)
             throw new \InvalidArgumentException('A message is required for ServerStreaming calls.');
         }
 
+        // Maintain forwards compatibility with older GAPIC clients not configured for REST server streaming
+        // @see https://github.com/googleapis/gax-php/issues/370
+        if (!$this->requestBuilder->pathExists($call->getMethod())) {
+            throw new BadMethodCallException('Streaming calls are not supported while using the REST transport');
```

Comment:
Alternatively, we can call the `HttpUnaryTransportTrait:startServerStreamingCall` and let it throw what it was doing before?

```php
use HttpUnaryTransportTrait {
    startServerStreamingCall as protected unsupportedServerStreamingCall;
}

$this->unsupportedServerStreamingCall($call, $options);
```

Reply:
🤯

Reply:
Totally Googled this because I wanted to know how to call a trait's method like a parent class' that was overloaded. Guess this is how one does that (`parent::` doesn't work because a `trait` isn't a class, of course).

Reply:
done!

Reply:
I didn't know you could change the visibility... that's pretty nice